tools/travis/build.sh

```diff
@@ -21,7 +21,7 @@ $ANSIBLE_CMD apigateway.yml
 
 cd $ROOTDIR
 
-./gradlew distDocker -x :core:swift3Action:distDocker -x :core:pythonAction:distDocker -PdockerImagePrefix=testing
```

Comment:
I don't understand what you mean, but OK
 you didn't modified gradle.build and travis still calls `./gradlew :tests:testLean`